### DISCUSSION TITLE
Yes/no question are not well done

### DISCUSSION THREAD

**Discussion by SanAlexis** [nyx-ei/pollingus#40] [General]
Hello @Pattykev,
It seems that we need to rewrite the yes/no question syntax. There should be a possibility to design this kind of question and avoid writing down the 2 possibilities: **yes** and **no**.

**Comment by Pattykev** [comment]
In this case, I think that we will leave the possibility to the recipient to answer and if it happens that his answer is not among the expected options (yes/no/n/y/t/f/0/1), we  should return an error.

**Reply by SanAlexis** [reply]
You don't get my point. I believe that we can create a **yes/no** question with a simplier syntax. Something like that:
```
### ~ Question content
```
With that automatically the program knows that it's a yes/no question. By this way we avoid to add the **2 answers**.

**Reply by Pattykev** [reply]
I've understood
